src/regtech_cleanup_api/entities/repos/filing_repo.py

```diff
@@ -1,8 +1,93 @@
 import logging
 
-
-from typing import TypeVar
+from typing import TypeVar, Any
+from sbl_filing_api.entities.models.dao import (
+    UserActionDAO,
+    FilingDAO,
+    SubmissionDAO,
+    ContactInfoDAO,
+)
+from sbl_filing_api.entities.repos import submission_repo
+from sqlalchemy import delete
+from sqlalchemy.ext.asyncio import AsyncSession
 
 logger = logging.getLogger(__name__)
 
 T = TypeVar("T")
+
+
+async def get_user_action_ids(
```

Comment:
We're moving away from all the async stuff.  Nargis's PR has updates to remove that, also, so follow along with that.  It's not huge for this, but maybe another story/PR to remove all this.

Reply:
This is being addressed in #31